src/main/resources/org/triplea/client/ui/javafx/fxml/TripleAMainMenu.fxml

```diff
@@ -128,22 +85,46 @@
                      <font>
                         <Font name="1942 report" size="13.0" />
                      </font>
+                     <effect>
+                        <DropShadow color="#00000080" offsetX="5.0" offsetY="10.0" />
+                     </effect>
                   </Button>
-                  <Button contentDisplay="TOP" layoutX="723.0" layoutY="85.0" mnemonicParsing="false" onAction="#showPbfPbemMenu" onMouseEntered="#startHover" onMouseExited="#endHover" text="%main.button.startPBEMPBF">
+                  <Button contentDisplay="TOP" layoutX="723.0" layoutY="85.0" mnemonicParsing="false" onAction="#showPlayByForumMenu" onMouseEntered="#startHover" onMouseExited="#endHover" prefWidth="168.0" text="%main.button.startPBF">
                      <graphic>
                         <ImageView fitHeight="150.0" fitWidth="150.0" pickOnBounds="true" preserveRatio="true">
                            <image>
-                              <Image url="@/org/triplea/client/ui/javafx/images/play_by_forum_email.png" />
+                              <Image url="@/org/triplea/client/ui/javafx/images/play_by_forum.png" />
                            </image></ImageView>
                      </graphic>
                      <font>
                         <Font name="1942 report" size="13.0" />
                      </font>
                      <HBox.margin>
-                        <Insets left="5.0" right="5.0" />
+                        <Insets bottom="5.0" left="5.0" right="5.0" top="5.0" />
+                     </HBox.margin>
+                     <effect>
+                        <DropShadow color="#00000080" offsetX="5.0" offsetY="10.0" />
+                     </effect>
+                  </Button>
+                  <Button contentDisplay="TOP" mnemonicParsing="false" onAction="#showPlayByEmailMenu" onMouseEntered="#startHover" onMouseExited="#endHover" prefWidth="168.0" text="%main.button.startPBEM">
+                     <graphic>
+                        <ImageView fitHeight="150.0" fitWidth="150.0" pickOnBounds="true" preserveRatio="true">
+                           <image>
+                              <Image url="@/org/triplea/client/ui/javafx/images/play_by_email.png" />
+                           </image>
+                        </ImageView>
+                     </graphic>
+                     <font>
+                        <Font name="1942 report" size="13.0" />
```

Comment:
any best practices out there to ensure all systems have this font? Is there a guarantee that all will?

Reply:
Using CSS classes :)
Will outsource such hardcoded values in a follow-up PR.